config/environments/production.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Settings here will take precedence over those in config/application.rb
 
+  # The forgery protection seems to false positive a lot around proxies...
+  config.action_controller.forgery_protection_origin_check = false
```

Comment:
> Disable forgery protection in production - not sure if this is controversial or not? 

This line does *not* disable forgery protection which would be bad. It only disables the origin check within the forgery protection check, which is OK.

The forgery protection token is still checked on all form submissions, which is good enough.

For discussion and related code on this see the PR that introduced this config. There's a mention of proxies: https://github.com/rails/rails/pull/22263